bindings/matrix-sdk-ffi/Cargo.toml

```diff
@@ -38,9 +38,36 @@ uniffi_macros = { workspace = true }
 tracing = { version = "0.1.29", default-features = false, features = ["log"] }
 android_logger = "0.11"
 log-panics = { version = "2", features = ["with-backtrace"]}
-matrix-sdk = { path = "../../crates/matrix-sdk", default-features = false, features = ["anyhow", "experimental-timeline", "e2e-encryption", "sled", "markdown", "experimental-sliding-sync", "socks", "rustls-tls"], version = "0.6.0" }
 
 [target.'cfg(not(target_os = "android"))'.dependencies]
 tracing = { workspace = true }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
-matrix-sdk = { path = "../../crates/matrix-sdk", features = ["anyhow", "experimental-timeline", "markdown", "experimental-sliding-sync", "socks"], version = "0.6.0" }
+
+[target.'cfg(target_os = "android")'.dependencies.matrix-sdk]
+path = "../../crates/matrix-sdk"
+default-features = false
+features = [
+    "anyhow",
+    "experimental-timeline",
+    "experimental-sliding-sync",
+    "markdown", 
+    "socks",
+    "e2e-encryption",
+    "sled",
+    "rustls-tls",
+]
+
+[target.'cfg(not(target_os = "android"))'.dependencies.matrix-sdk]
+path = "../../crates/matrix-sdk"
+default-features = false
+features = [
+    "anyhow",
+    "experimental-timeline",
+    "markdown",
+    "experimental-sliding-sync",
+    "socks",
+    "e2e-encryption",
+    "automatic-room-key-forwarding",
+    "sled",
+    "native-tls",
```

Comment:
If both lists of features for `matrix-sdk` could be alphabetically sorted, it could help to see the difference between the features :-).

Reply:
Sorted: https://github.com/matrix-org/matrix-rust-sdk/pull/1365/commits/da77269f5bd427cc940c5376ae55026f54729cc8